server/proxy/config.ini

```diff
@@ -18,19 +18,14 @@ Port = 3389
 Mouse = 1
 Keyboard = 1
 
-[Graphics]
-GFX = 1
-BitmapUpdate = 1
```

Comment:
we don't want that one anymore?

Reply:
nope, gfx moved to channels section and we never used BitmapUpdate value